arch/cpu/cc26x0-cc13x0/rf-core/ble-hal/rf-ble-cmd.c

```diff
@@ -45,6 +45,7 @@
 #include "rf_ble_cmd.h"
 #include "rf-core/rf-core.h"
 #include "rf-core/ble-hal/rf-ble-cmd.h"
+#include <inttypes.h>
```

Comment:
If this file is missing, shouldn't the include be in `rf_ble_cmd.h` instead? There are plenty of uses of `uint8_t` in the function prototypes.

Reply:
iirc, it was the `PRIX32` on line 525 that I got a complaint about. I am guessing `stdint.h` is carried with some of the other includes in `rf_ble_cmd.h` and that is why it passes. But in any case, you are right -  I added `stdint.h` to the header-file (as opposed to `inttypes.h`) to keep it as targeted as possible.

Reply:
Oh, for `PRIX32` you are right about the header, nevermind my comment then.